rules/windows/command_and_control_common_webservices.toml

```diff
@@ -0,0 +1,76 @@
+[metadata]
+creation_date = "2020/11/04"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/04"
+
+[rule]
+author = ["Elastic"]
+description = """
+Adversaries may implement command and control communications that use common web services in order to hide their
+activity. This attack technique is typically targeted to an organization and uses web services common to the victim network which allows the adversary to blend into legitimate traffic.
+activity. These popular services are typically targeted since they have most likely been used before a compromise and
+allow adversaries to blend in the network.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Connection to Commonly Abused Web Services"
+risk_score = 21
+rule_id = "66883649-f908-4a5b-a1e0-54090a1d3a32"
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Command and Control"]
+type = "eql"
+
+query = '''
+network where network.protocol == "dns" and
+             /* Add new WebSvc domains here */
+              wildcard(dns.question.name, "*.githubusercontent.*",
+                                          "*.pastebin.*",
+                                          "*drive.google.*",
+                                          "*docs.live.*",
+                                          "*api.dropboxapi.*",
+                                          "*dropboxusercontent.*",
+                                          "*onedrive.*",
+                                          "*4shared.*",
+                                          "*.file.io",
+                                          "*filebin.net",
+                                          "*slack-files.com",
+                                          "*ghostbin.*",
+                                          "*ngrok.*",
+                                          "*portmap.*",
+                                          "*serveo.net",
+                                          "*localtunnel.me",
+                                          "*pagekite.me",
+                                          "*localxpose.io"
```

Comment:
Hey @Samirbous,

I saw this one ([notabug.org](https://www.virustotal.com/gui/domain/notabug.org/relations)) in the wild the other day and it ended up on this [write-up](https://blog.malwarebytes.com/cybercrime/2020/10/fake-covid-19-survey-hides-ransomware-in-canadian-university-attack/).  Might be worth to add to this already good list.

Reply:
nice!